lib/routes/airlevel/levelrank.ts

```diff
@@ -0,0 +1,69 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+import { load } from 'cheerio'; // 类似 jQuery 的 API HTML 解析器
+
+export const route: Route = {
+    path: ['/rank/:status', '/rank'],
+    radar: [
+        {
+            source: ['m.air-level.com/rank/:status', 'm.air-level.com/rank'],
+            target: '/rank/:status',
+        },
+    ],
+    parameters: {
+        status: '地区',
+    },
+    name: '空气质量排行',
+    categories: ['popular'],
+    maintainers: ['lt'],
+    example: 'rank/best,rank',
+    handler,
+};
+
+async function handler(ctx) {
+    const status = ctx.req.param('status');
+    const currentUrl = `https://m.air-level.com/rank`;
```

Comment:
[Unnecessary template literals](https://docs.rsshub.app/joinus/advanced/script-standard#string). Only use it when necessary.